.github/workflows/pr_review.yml

```diff
@@ -30,8 +30,8 @@ jobs:
           OPENAI_API_KEY: ${{ secrets.OPENAI_API_KEY }}
         with:
           debug: true
-          review_simple_changes: false
-          review_comment_lgtm: false
+#          review_simple_changes: false
+#          review_comment_lgtm: false
           max_files: 1
           disable_release_notes: true
           openai_retries: 1
```

Comment:
The value of openai_retries is currently set to 1. It may be beneficial to increase the number of retries for better reliability.